site/en/blog/hardware-accelerated-animations/index.md

```diff
@@ -0,0 +1,135 @@
+animations will likely come up. But what do these even mean? Hardware-accelerated styles are those
+that leverage the GPU (Graphical Processing Unit) rather than the CPU (Central Processing Unit) to
+render visual styles. This is because the GPU can render visual changes on a web page faster than
+the CPU. 
+
+Using the GPU to offload graphic-intensive transitions and animations means smoother visuals and
+less jank, as these animations are not affected by the main thread. By pulling them off the main
+thread, your animations bypass the crust of other active scripts running on your page, which would
+slow down your visuals and create jank. 
+
+## Enabling hardware-accelerated animations
+
+Hardware-accelerated animations are composited as layers and help developers achieve buttery-smooth
+60 FPS (frames-per-second) animations to improve visual rendering performance. There are currently a
+few ways to specify and enable hardware-accelerated animations and transitions on the web:
+
+- Use CSS `transform` functions or transition the `opacity` or `filter` values
+- Add the `will-change` property to your element.
+- Create an animated canvas drawing via `OffscreenCanvas`
+- Create a WebGL 3D drawing
+
+<div class="aside aside--note"> The Chromium rendering team is continually
+<a href="https://chromestatus.com/metrics/css/animated">tracking</a> usage of the most-animated properties to
+determine what should be next in regard to enabling hardware acceleration. While the current CSS
+properties that are hardware-accelerated by default only include <code>opacity</code>, <code>filter</code>, and
+<code>transform</code> for now, <code>background-color</code> and <code>clip-path</code> will soon join the list. </div>
+
+What else is becoming hardware accelerated by default in Chromium? There are a few things coming
+down the pipeline, including SVG animations, something that developers have been keenly
+[requesting](https://codepen.io/chrisgannon/full/WvJMXP).
+
+## Hardware accelerated SVG animations
+
+SVG is a great addition to any website, and now those interactions with SVG can be more performant.
+As of Chromium 89, Chrome will join the likes of Firefox to enable hardware-acceleration by
+default on SVG animations. What do you, the developer, need to do? Nothing—this will be
+automatically applied for SVG animations in Chromium 89+.
+
+### Example
+
+Let's take a look at the differences between an SVG animation with and without hardware acceleration turned on. Loading indicators are commonly-used UI elements, such as this one seen on facebook.com. These indicators hint at work being done on the server, while the user waits for a response. In the case shown here, the response would be to load additional results in the sidebar.
+
+<figure>
+  {% img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/CBSrDDfkO8Ab693jU50K.gif", alt="Facebook sidebar UI", width="305", height="384" %}
```

Comment:
Can you convert this to GIF? I know it's extra work but it's important to practice what we preach https://web.dev/replace-gifs-with-videos/